### PR TITLE
Add .crypto resolution for Verified Twitter

### PR DESCRIPTION
**Pull request by sudoryan**
Shows if a .crypto domain is Twitter Verified for resolution.
Test with ijustwannatestsomething2.crypto sending ETH.

Example

### CODE REVIEW

File: src/directives/AddrResolver.js [hunk 1]
```diff
@@ -256,38 +268,53 @@ const AddrResolver = {
...
+                messageDiv.appendChild(twitterVerifiedPar);
+              }
+            }
+            appendElement(messageDiv);
           }
         } catch (err) {
           _this.isValidAddress = false;
           _this.hexAddress = '';
           _this.avatar = '';
+          console.log(err);
```
Comment (gamalielhere) [inline src/directives/AddrResolver.js:317]: Please remove console

**Review by gamalielhere — CHANGES_REQUESTED** [review]
Everything looks good, just need to remove the console and the yarn.lock

File: src/directives/AddrResolver.js [hunk 2]
```diff
@@ -256,38 +268,53 @@ const AddrResolver = {
...
+              const twitterUsername = await resolution.cns
+                .twitter(domain)
+                .catch(() => null);
+              if (twitterUsername) {
+                const twitterVerifiedPar = document.createElement('p');
+                twitterVerifiedPar.classList.add('twitter-verify');
+
+                twitterVerifiedPar.innerHTML = `<div style="display:flex; align-items:center; padding: 0 0 1em 1em"><img style="padding: 0 6px 0 0"src="${twitterVerifiedLogo}" /> <a href="https://twitter.com/${twitterUsername}" target="_blank" style="margin-right:5px; font-weight:600">@${twitterUsername}</a> - ${_this.$t(
+                  'ens.unstoppableResolution.twitter-verified'
+                )} <a href="https://chain.link/" target="_blank" style="margin-left: 5px">Chainlink</a></div>`;
```
Comment (kvhnuke) [inline src/directives/AddrResolver.js:307]: add `no referrer`

**Review by kvhnuke — CHANGES_REQUESTED** [review]

**Review by kvhnuke — APPROVED** [review]

**Comment by gamalielhere** [comment]
Included #2624

**Pull request closed without merging**